setup.py

```diff
@@ -0,0 +1,107 @@
+"""A setup script for APEL.
+
+This script installs the APEL library, client, parsers and server. This
+should be similar to installing the RPMs for apel-lib, apel-client,
+apel-parsers, and apel-server, although there may be some differences.
```

Comment:
Does this not install the .pyc/o like the SSM setup.py? If so, should use a similar comment to SSM's.